.eslintrc.cjs

```diff
@@ -9,12 +9,6 @@ module.exports = {
   plugins: ['jest'],
   rules: {
     'no-debugger': 'error',
-    'no-unused-vars': [
-      'error',
-      // we are only using this rule to check for unused arguments since TS
-      // catches unused variables but not args.
```

Comment:
Add `noUnusedParameters` in tsconfig for checking unused args.

Reply:
If we add this configuration option, it will lead to numerous errors due to a large number of unused parameters. I believe this is the reason why we have configured `args: 'none'`, as we don't want to check for unused parameters. Therefore, we don't need to add this configuration option, and we also don't need to set `args: 'none'` because TypeScript doesn't by default check for unused parameters, for example [`defineExpose`](https://github.com/vuejs/core/blob/8998afa42755cbdb3403cd6c0fe158980da8492c/packages/runtime-core/src/apiSetupHelpers.ts#L173)